test/common/test_executor.cpp

```diff
@@ -0,0 +1,123 @@
+/*
+ * SPDX-License-Identifier: BSD-3-Clause
+ *
+ *  Point Cloud Library (PCL) - www.pointclouds.org
+ *  Copyright (c) 2014-, Open Perception, Inc.
+ *  Author(s): Shrijit Singh <shrijitsingh99@gmail.com>
+ *
+ */
+
+#include <pcl/experimental/executor/executor.h>
+#include <pcl/test/gtest.h>
+
+using namespace executor;
+
+using ExecutorAlwaysAvailableTypes = ::testing::Types<const default_inline_executor,
+                                                      default_inline_executor,
+                                                      const default_sse_executor,
+                                                      default_sse_executor>;
+
+using ExecutorCondAvailableTypes = ::testing::Types<const default_inline_executor,
+                                                    default_inline_executor,
+                                                    const default_sse_executor,
+                                                    default_sse_executor,
+                                                    const default_omp_executor,
+                                                    default_omp_executor>;
+
+using ExecutorTypes = std::conditional_t<is_executor_available_v<omp_executor>,
```

Comment:
When does this switch happen? Does any CI run with OMP disabled?

Reply:
If you run the test locally and don't have OMP.